src/main/g8/src/main/scala/$package$/Main.scala

```diff
@@ -0,0 +1,19 @@
+package $package$
+
+import java.io.IOException
+
+import zio.{ App, ExitCode, URIO, ZEnv, ZIO }
+import zio.console.{ getStrLn, putStrLn, Console }
+
+object Main extends App {
+
+  val app: ZIO[Console, IOException, Unit] =
+    for {
```

Comment:
Let's use the new syntax and drop the curly braces

Reply:
Removed { in for and Main object